server/api/fileOperations.js

```diff
@@ -0,0 +1,123 @@
+// @flow
+import Router from "koa-router";
+import { NotFoundError, ValidationError } from "../errors";
+import auth from "../middlewares/authentication";
+import { FileOperation, User, Collection, Team } from "../models";
+import policy from "../policies";
+import { presentFileOperation } from "../presenters";
+import { getSignedUrl } from "../utils/s3";
+import pagination from "./middlewares/pagination";
+
+const { authorize } = policy;
+const router = new Router();
+
+router.post("fileOperations.info", auth(), async (ctx) => {
+  const { id } = ctx.body;
+  ctx.assertUuid(id, "id is required");
+  const user = ctx.state.user;
+  const team = await Team.findByPk(user.teamId);
+
+  const fileOperation = await FileOperation.findByPk(id, {
+    include: [
+      {
+        model: User,
+        as: "user",
+        paranoid: false,
+      },
+      {
+        model: Collection,
+        as: "collection",
+        paranoid: false,
+      },
+    ],
```

Comment:
Seems like this is repeated a lot, we can use sequelize's `defaultScope` in the model to make sure they are always included and remove this include statement in several places